values.yaml

```diff
@@ -48,8 +48,8 @@ image:
 #  - name: regcred
 imagePullSecrets: []
```

Comment:
Justin - what is `imagePullSecrets` for?

Reply:
Its a config option that Kubernetes exposed that enables using containers that are hosted in private registries:

https://kubernetes.io/docs/concepts/containers/images/#specifying-imagepullsecrets-on-a-pod

Reply:
![image](https://github.com/armory-io/remote-network-agent-helm-chart/assets/711726/58b8dc10-788c-46a0-b39c-ea691c405a9d)

Reply:
Lastly this link might be most relevant and is how I was introduced to the concept of imagePullSecrets:
https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/